src/commands/Fun/destruct.js

```diff
@@ -0,0 +1,64 @@
+exports.run = function(bot,msg,args) {
+
+    let parsedArgs = bot.utils.parseArgs(args, ['d:','s:']);
+
+    if(parsedArgs.leftover.length < 1) {
+        throw 'Please provide a secret message';
+    }
+
+    let text = parsedArgs.leftover;
+    let message = text.join(' ');
+    let delay = isNaN(parsedArgs.options.d) ? 5000 : parsedArgs.options.d;
+    let style = (typeof parsedArgs.options.s === 'string') ? parsedArgs.options.s : 'plain';
+
+    msg.delete();
+
+    switch(style) {
+    case 'embed':
+        message = bot.utils.embed(
```

Comment:
If you change this to

```
message = {
    embed: bot.utils.embed(...)
};
```

Then you can just move the msg.channel.send call out of the switch statement and that way you'll stop repeating yourself.